### PR TITLE
Home Page APIs

### PR DESCRIPTION
**Pull request by nkhl2811**
Home Page APIs added to backend.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -27,9 +27,10 @@ app.get('/', (req, res) => {
 });
 
 app.use('/auth', require('./routes/auth'));
+app.use('/homepage',require('./routes/auth'));
 
 // set port, listen for requests
-const PORT = process.env.PORT || 8080;
+const PORT = process.env.PORT || 3000;
```
Comment (ishantgupta777) [inline app.js:33]: Change the port or use the environment varables

File: config/db.config.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 const Sequelize = require('sequelize');
 
 const dbConfig = {
-  HOST: 'localhost',
+  HOST: '127.0.0.1',
   USER: 'root',
-  PASSWORD: '',
+  PASSWORD: 'password',
```
Comment (ishantgupta777) [inline config/db.config.js:6]: Change the localhost and password

File: models/User.js [hunk 3]
```diff
@@ -12,8 +12,8 @@ module.exports = (sequelize, Sequelize) => {
       lastname: { type: Sequelize.STRING(20), allowNull: false },
       username: { type: Sequelize.STRING(20), defaultValue: null },
       email: { type: Sequelize.STRING(50), allowNull: false },
-      password: { type: Sequelize.STRING(100), allowNull: false },
-      usertype: { type: Sequelize.CHAR(2), allowNull: false },
+      password: { type: Sequelize.STRING, allowNull: false },
+      usertype: { type: Sequelize.STRING(2), allowNull: false },
```
Comment (ishantgupta777) [inline models/User.js:16]: Revert these changes

File: routes/auth.js [hunk 4]
```diff
@@ -1,8 +1,23 @@
...
 
 router.post('/login', user.login);
 
+router.get('/events',verifyToken, homepage.displayEvents);
+
+
+function verifyToken(req, res, next) {
+
+    const user_token = req.headers['Token'];
+    if (typeof token !== 'undefined') {
```
Comment (ishantgupta777) [inline routes/auth.js:15]: Change the token to user_token

**Comment by nkhl2811** [comment]
Changes done
